### DISCUSSION TITLE
v23.02.08

### DISCUSSION THREAD

**Discussion by Tokey** [timdrevitch/killdeer-prey#3] [General]
- This is updated up to AI that follows the player.
- The DevSandbox Map is dead.
- The project has items and nest bp. It currently is not in the scene. 
- Spawn point can be set.
- Starting from testmap.

<hr /><em>This discussion was created from the release <a href='https://github.com/timdrevitch/killdeer-prey/releases/tag/'>v23.02.08</a>.</em>

**Comment by timdrevitch** [comment]
Sweet! Thanks!
